openedx/core/djangoapps/course_groups/models.py

```diff
@@ -185,14 +185,35 @@ class CourseCohortsSettings(models.Model):
     @property
     def cohorted_discussions(self):
         """Jsonify the cohorted_discussions"""
-        return json.loads(self._cohorted_discussions)
+
+        segmented_discussions = self.segmented_discussions.all()
```

Comment:
Will this be filtered on the ID?  I am still a little new with the Django ORM, but I do not see a filter being applied to this all.